ui/tests/integration/components/clients/config-test.js

```diff
@@ -64,7 +64,6 @@ module('Integration | Component | client count config', function (hooks) {
     this.createModel('disable');
 
     await render(hbs`
-      <div id="modal-wormhole"></div>
```

Comment:
I think this was talked about in one of the threads in our channel, but I can't seem to remember. With the adoption of hds modals, will all the `modal-wormhole` be removed in tests?

Reply:
Yep! Modals are renders in place, inside a conditional, so there's no "other" location for them! I was going to remove these divs all at the end, but figured if I'm updating tests I might as well remove them as I go